app/models/character_attribute.rb

```diff
@@ -1,5 +1,7 @@
 class CharacterAttribute < ApplicationRecord
-  belongs_to :character_type
+  belongs_to :character_type, class_name: "CharacterType", optional: true, inverse_of: :character_attributes
```

Comment:
@StefanGajic why do you need `class_name`? Rails should be able to infer it from `character_type`.

Reply:
I understand, I was looking at different examples and some of them had `class_name` in the model and I guess I forgot to delete it. I can see from docs that :
"If the name of the other model cannot be derived from the association name, you can use the :class_name option to supply the model name. For example, if a book belongs to an author, but the actual name of the model containing authors is Patron, you'd set things up this way: `belongs_to :author, class_name: "Patron"` "